pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -5611,6 +5611,36 @@ func TestImportPgDump(t *testing.T) {
 		sqlDB.Exec(t, "IMPORT PGDUMP ($1)", srv.URL)
 		sqlDB.CheckQueryResults(t, `SELECT * from t`, [][]string{{"2", "42", "1"}, {"4", "42", "3"}})
 	})
+
+	t.Run("more-target-cols-than-data", func(t *testing.T) {
+		data := `
+COPY public.t (id, link, lang, feed, date) FROM stdin;
+4964055	http://foo	france	http://bar	2021-03-16 05:46:23.239227
+\.
+
+INSERT INTO public.t (id, link, lang, feed, date) VALUES (4964056, 'http://foo', 'france', 'http://bar', '2021-03-16 05:46:23.239227');
+			`
+		srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			if r.Method == "GET" {
+				_, _ = w.Write([]byte(data))
+			}
+		}))
+		defer srv.Close()
+		defer sqlDB.Exec(t, "DROP TABLE t")
+		sqlDB.Exec(t, `
+IMPORT TABLE t (
+id INT8 NOT NULL PRIMARY KEY DEFAULT unique_rowid(),
+feed STRING,
```

Comment:
nit: I'd use generic names for these like a, b, c ,... and values like 123, 'a', 'b' above/below.

Reply:
done, simplified the test case.